packaging/language/bower.py

```diff
@@ -161,6 +181,7 @@ def main():
         offline=dict(default='no', type='bool'),
         production=dict(default='no', type='bool'),
         path=dict(required=True),
+        relative_execpath=dict(default=False, required=False),
```

Comment:
In current 2.0 we've added a type='path' that does expanduser and expandvars automatically on the parameter.  might that be better than just using generic str type?